### DISCUSSION TITLE
Freshwater 50

### DISCUSSION THREAD

**Discussion by bobbywaz** [TheRealFalseReality/aquapi#76] [Q&A] [answered]
Hey, I've already got an established HA, and I've got a few ESP32's lying around (the right one), and a 3d printer. I'm just trying to get a balanced freshwater 50 gallon that I can monitor remotely and get alerts. I'm just interested in temp, water level, and I'm not sure what else I should be monitoring, I'm not trying to spend $1k on a 400gal salt water tank, I got the tank for $55... but also I haven't had a tank in a while and I'm not sure what's worth monitoring or just using test kit once in a while for? Could you give me a little direction?

**Comment by TheRealFalseReality** [comment, marked as answer]
Hello! You may only need pH but even then, not really important at all in a freshwater tank.
